src/items/class-features/bedside_manner_(ex).json

```diff
@@ -0,0 +1,115 @@
+{
+  "_id": "TnDwezeK41boR3kD",
+  "name": "Bedside Manner (Ex)",
+  "type": "feat",
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "",
+    "ability": "",
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "none",
+      "condition": "",
+      "cost": 0
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "none",
+      "value": null
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<p>&nbsp;@Icon[type:language-dependent] Language-dependent @Icon[type:mind-affecting] Mind-Affecting @Icon[type:sense-dependent] Sense-Dependent</p>\n<p>When you make a Medicine check to treat deadly wounds or successfully aid another for another creature’s Medicine check to treat deadly wounds, if the Medicine check is successful you can add your Charisma bonus to the number of Hit Points recovered as a result of the skill check. Additionally, when you make a Medicine check to treat disease or treat drugs or poison, or successfully aid another for another creature’s Medicine check to do so, if the Medicine check is successful, the saving throw bonus granted by the skill checks increases to +6. If the associated saving throw succeeds, it counts as two consecutive successes for the purposes of curing the intended affliction.</p>"
```

Comment:
Let's have each descriptor in its own on a new line (in its own `<p>` paragraph). Make sure to capitalise the `dependent` in language-dependent to ensure consistency with the other descriptors

Reply:
I think currently they don't have labels, they just have the image. But since we're adding labels they should be on different lines. An audit of existing improvs should be done (for both descriptors in text, and ticked on the descriptors tab), but it doesn't have to be part of this PR